openeogeotrellis/geopysparkdatacube.py

```diff
@@ -93,6 +93,12 @@ def filter_temporal(self, start, end) -> 'GeopysparkCubeMetadata':
     def temporal_extent(self) -> tuple:
         return self._temporal_extent
 
+    @property
+    def opensearch_link_titles(self) -> List[str]:
+        """Get opensearch_link_titles from band dimension"""
+        names_with_aliases = zip(self.band_dimension.band_names, self.band_dimension.band_aliases)
```

Comment:
you also might add a prefix like `internal:` to the band name aliases in `layercatalog.json` so you can more robustly detect the internal band names (instead of assuming the first alias is the internal name)

Reply:
But then we do need to strip it off again right? When exposing externally.
Will merge for now, to be able to use this on Creo.

Reply:
stripping off is not necessary when exposing externally, I think, 
it might even be better with prefix exposed  as it indicates that it is a special name for internal use, not for end users